Library/Homebrew/test/cask/cask_spec.rb

```diff
@@ -14,11 +14,11 @@
             ["1.2.2", "1001"],
           ])
 
+          # Installed caskfile must exist to count as installed.
+          allow_any_instance_of(Pathname).to receive(:exist?).and_return(true)
```

Comment:
Would it be possible to scope this down a little bit by maybe specifying the number of times we expect it to be called?

Reply:
Does that work with `allow_*` methods?

Reply:
Doesn't look like it. I also see that this seems to be a common pattern in our test suite so ignore what I said.

It might be possible with `expect_any_instance_of` but that doesn't seem to be used in our codebase at all.